src/utils/getGuildMemberDetails.ts

```diff
@@ -0,0 +1,27 @@
+import { INTERNAL_SERVER_ERROR } from "../constants/responses";
+import { DISCORD_BASE_URL } from "../constants/urls";
+import { env } from "../typeDefinitions/default.types";
+import { discordMemberDetails } from "../typeDefinitions/discordMessage.types";
+
+export async function getGuildMemberDetails(
```

Comment:
why provide the whole env to the function when we are just using the bot token and guild id? Please pass just the bot token as a parameter.